### PR TITLE
fix: init `publicPath` to undefined

### LINKED ISSUE #158: regression: publicPath can't be false
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

The webpack config validator does not allow publicPath: false. undefined is allowed, but not false. This change thus makes it impossible to cause __webpack_public_path__ to be used.

**If the current behavior is a bug, please provide the steps to reproduce.**

See: https://github.com/webpack-contrib/file-loader/pull/145

**What is the expected behavior?**

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**

### PR DESCRIPTION
**Pull request by d3viant0ne**
**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**

**If relevant, did you update the README?**

**Summary**
Initializing `publicPath` to false was preventing the new option from being used with `webpack-config-validator
`

See comment in #145

**Does this PR introduce a breaking change?**

No

**Other information**

Fixes #158

### CODE REVIEW

**Comment by d3viant0ne** [comment]
It already has two to cover the feature in question, just changing how we init publicPath because of the issue mentioned in the PR that broke it.

**Comment by bebraw** [comment]
Ok, cool. JS. 👍

File: index.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ module.exports = function(content) {
 	}
 
 	var publicPath = "__webpack_public_path__ + " + JSON.stringify(url);
-	if (config.publicPath !== false) {
+	if (config.publicPath !== undefined) {
```
Comment (michael-ciniawsky) [inline index.js]:
```js
if (!config.publicPath) {}
```

Maybe even 'safer', but enterily a nitpick

**Comment by jcperez-ch** [comment]
Any news regarding this bug?

**Comment by michael-ciniawsky** [comment]
@d3viant0ne  @bebraw Is there anything left to do here/blocking this ? :)

**Comment by bebraw** [comment]
@michael-ciniawsky Not that I know of.

**Comment by michael-ciniawsky** [comment]
Please semver patch someone when time :)

**Pull request merged**